### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by RemyaManoharan**

### CODE REVIEW

File: javascript/javascript1/week3/notes.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const notes = [];
+// function to push objects to array
+function saveNote(content, id) {
+  const note = { id: id, content: content };
```
Comment (dinythomas89) [inline javascript/javascript1/week3/notes.js:4]: In ES6 you have this short hand property for object. So you can write as
``` 
const note = {
id,
content,
}
```
https://javascript.plainenglish.io/object-literals-using-object-property-shorthand-6360825c60ef

File: javascript/javascript1/week3/phoneuse.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const activities = [];
+
+function addActivity(date, activity, duration) {
+  const items = { date: date, activity: activity, duration: duration };
```
Comment (dinythomas89) [inline javascript/javascript1/week3/phoneuse.js:4]: same here as above

**Review by dinythomas89 — APPROVED** [review]
When you use file names with 2 or more words try to separate it with hyphen, underscore or camelcase as it will it easier to read

File: javascript/javascript1/week3/phoneuse.js (hunk 2 above)
Comment (RemyaManoharan) [reply javascript/javascript1/week3/phoneuse.js:4]: Thanks for the review.

**Pull request merged**